code/src/components/TodoInput.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import todos from '../reducers/todos';
+import { InputContainer, ItemInput, AddItemButton } from 'styling/TodoInputStyling';
```

Comment:
It is a good practice to write them as they get more and more with imports and styled components.

import {
   InputContainer, 
   ItemInput, 
   AddItemButton 
} from 'styling/TodoInputStyling';